src/rez/backport/shutilwhich.py

```diff
@@ -1,64 +1,80 @@
-import os
-import os.path
-import sys
+# Modified version of whichcraft, a shim of `shutil.which`.
+# 'env' environ dict override has been added by Colin Kennedy.
+
+# Source: https://github.com/cookiecutter/whichcraft/blob/0.6.1/whichcraft.py
+# Copyright (c) 2015-2016, Daniel Roy Greenfeld
+# All rights reserved.
 
```

Comment:
Not sure if this is okay, since Rez is going to be re-licensed as Apache2.0 (#1119) and the [`whichcraft`](https://github.com/cookiecutter/whichcraft) is under BSD-3.